.github/workflows/testing_gsoc.yml

```diff
@@ -94,7 +92,7 @@ jobs:
         ; done)
 
     - name: Collect coverage
-      if: ${{ success() && inputs.event_name == 'push' && inputs.branch_name == 'develop' && inputs.xdist == 'no'}}
+      if: ${{ success() && inputs.event_name == 'push' && inputs.branch_name == 'develop' }}
```

Comment:
we also don't have "inputs" parameters, as this is not called as function, we don't use testing.yml as base